### PR TITLE
Corrected test error in mailwatch_geoip_update.php

### PR DESCRIPTION
**Pull request by stefaweb**

### CODE REVIEW

File: tools/Cron_jobs/mailwatch_geoip_update.php [hunk 1]
```diff
@@ -170,7 +170,7 @@
 
         // Apply MailWatch rights on files from the last run
         $mwUID =  exec('cat /etc/sudoers.d/mailwatch | grep "User_Alias MAILSCANNER" | sed "s/.*= \(.*\).*/\1/"', $retval_cat);
-        if ($retval_cat > 0) {
```
Comment (endelwar) [inline tools/Cron_jobs/mailwatch_geoip_update.php]: $retval_cat is an INT, why do you check as an array?
http://php.net/manual/en/function.exec.php

Comment (stefaweb) [inline tools/Cron_jobs/mailwatch_geoip_update.php]: I've an error with `$retval_cat > 0`. The system say that the result is an array.

Comment (endelwar) [inline tools/Cron_jobs/mailwatch_geoip_update.php]: ok, the second parameter is missing ($output) which is an array, $retval_cat must be the third

Comment (endelwar) [inline tools/Cron_jobs/mailwatch_geoip_update.php]:
```php
$mwUID =  exec('cat /etc/sudoers.d/mailwatch | grep "User_Alias MAILSCANNER" | sed "s/.*= \(.*\).*/\1/"', $output_cat, $retval_cat);
```

**Pull request merged**